### PR TITLE
Distribution metrics guide updates

### PR DESCRIPTION
**Pull request by andrewardito**
### What does this PR do?
Add terraform section to distribution metrics guide

### Preview
https://docs-staging.datadoghq.com/andrew.ardito/distributionmetricsupdates/tracing/guide/ddsketch_trace_metrics/
### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/tracing/guide/ddsketch_trace_metrics.md [hunk 1]
```diff
@@ -41,9 +41,37 @@ The APM Service and Resource pages use this metric type automatically. This mean
 - Distribution Metrics now power APM Service and Resource pages as Datadog rolls this out for more customers.
 - Datadog has migrated all manual queries from dashboards and monitors to the new metrics automatically on your behalf, while retaining the existing metrics to support historical views.
 
+**I'm using Terraform, what does this change mean for me?**
+- The existing metrics are not going away, this means that your Terraform definitions are still in place and still work well.
```
Comment (kayayarai) [inline content/en/tracing/guide/ddsketch_trace_metrics.md:45]:
```suggestion
The existing metrics are not going away; your Terraform definitions are still in place and still work.
```

File: content/en/tracing/guide/ddsketch_trace_metrics.md [hunk 2]
```diff
@@ -41,9 +41,37 @@ The APM Service and Resource pages use this metric type automatically. This mean
...
+- We recommend changing these to use the new DDSketch-based metrics that offer [better precision][4].
```
Comment (kayayarai) [inline content/en/tracing/guide/ddsketch_trace_metrics.md:46]:
```suggestion
 To take advantage of the [better precision][4] offered by the new DDSketch-based metrics, change your Terraform definitions as shown in the following examples.
```

File: content/en/tracing/guide/ddsketch_trace_metrics.md [hunk 3]
```diff
@@ -41,9 +41,37 @@ The APM Service and Resource pages use this metric type automatically. This mean
...
+- See the below examples for how to update your terraform definitions.
```
Comment (kayayarai) [inline content/en/tracing/guide/ddsketch_trace_metrics.md:47]:
```suggestion

```

File: content/en/tracing/guide/ddsketch_trace_metrics.md [hunk 4]
```diff
@@ -41,9 +41,37 @@ The APM Service and Resource pages use this metric type automatically. This mean
 - Distribution Metrics now power APM Service and Resource pages as Datadog rolls this out for more customers.
 - Datadog has migrated all manual queries from dashboards and monitors to the new metrics automatically on your behalf, while retaining the existing metrics to support historical views.
 
+**I'm using Terraform, what does this change mean for me?**
```
Comment (kayayarai) [inline content/en/tracing/guide/ddsketch_trace_metrics.md:44]:
```suggestion
**I'm using Terraform. What does this change mean for me?**
```

**Review by kayayarai — CHANGES_REQUESTED** [review]
Some edits for you

File: content/en/tracing/guide/ddsketch_trace_metrics.md (hunk 1 above)
Comment (omris1) [reply content/en/tracing/guide/ddsketch_trace_metrics.md]: @andrewardito how did I miss this? it's not "the existing metrics are not going away". I updated this in my draft to be "The existing metrics are still around"

**Review by kayayarai — APPROVED** [review]

**Pull request merged**